### PR TITLE
community header: Fix missing import

### PR DESCRIPTION
**Pull request by jrcastro2**
* Replaces semantic ui react Image with
  Image of react-invenio-forms

### CODE REVIEW

File: src/lib/components/Communities/CommunityHeader.js [hunk 1]
```diff
@@ -32,8 +34,11 @@ class CommunityHeaderInfo extends React.Component {
     const logoLink = community ? community?.links?.logo : imagePlaceholderLink;
     return (
       <Container className="community-header">
-        <Image size="mini" src={logoLink} /> {community?.metadata?.title}
-        <CommunityHeaderButton />
+        <Image className="community-logo-header" src={logoLink} />
```
Comment (zzacharo) [inline src/lib/components/Communities/CommunityHeader.js:37]: you miss the fallback url e.g point to `/static/images/sqaure-placeholder.png`?

File: src/lib/components/Communities/CommunityHeader.js [hunk 2]
```diff
@@ -32,8 +34,11 @@ class CommunityHeaderInfo extends React.Component {
...
+        <div className='mt-10'>
+          {community?.metadata?.title}
```
Comment (zzacharo) [inline src/lib/components/Communities/CommunityHeader.js:39]: Creating a community I think requires a title. Can you check please and if it's the case then you should use `community.metadata.title` :)

File: src/lib/components/Communities/CommunityHeader.js (hunk 1 above)
Comment (jrcastro2) [reply src/lib/components/Communities/CommunityHeader.js]: Right, I was actually checking for the fallback above with:
```
const logoLink = community ? community?.links?.logo : imagePlaceholderLink;
```
But is not anymore needed.

File: src/lib/components/Communities/CommunityHeader.js (hunk 2 above)
Comment (jrcastro2) [reply src/lib/components/Communities/CommunityHeader.js]: Right, it's a leftover. Thanks!

**Review by ntarocco — APPROVED** [review]

**Pull request merged**